js/src/functions.js

```diff
@@ -2297,33 +2297,18 @@ Functions.previewSql = function ($form) {
         sep + 'preview_sql=1' +
         sep + 'ajax_request=1';
     var $messageBox = Functions.ajaxShowMessage();
+    var modal = $('#previewSqlModal');
+    modal.modal('show');
```

Comment:
Could you please rebase this one and remove the non needed `modal` variables please ?
```suggestion
    $('#previewSqlModal').modal('show');
```